### ISSUE TITLE
Command line cannot play certain sounds

### ISSUE THREAD

**Issue by thetacola** [N3ther/Algonquin-TTS#1]
Essentially, any character that is not in base latin (ʈ, ɛ, etc.) does not work when run from command line, but does when run via Eclipse.

**Comment by thetacola** [comment]
Exception when playing Unicode:

```
Exception in thread "main" java.lang.IllegalArgumentException: could not play 'null'
        at net.oijon.algonquin.App.createAudio(App.java:104)
        at net.oijon.algonquin.App.main(App.java:120)
Caused by: java.io.IOException: Stream closed
        at java.base/java.io.BufferedInputStream.getInIfOpen(BufferedInputStream.java:157)
        at java.base/java.io.BufferedInputStream.fill(BufferedInputStream.java:244)
        at java.base/java.io.BufferedInputStream.read(BufferedInputStream.java:263)
        at java.desktop/com.sun.media.sound.RIFFReader.read(RIFFReader.java:126)
        at java.desktop/com.sun.media.sound.RIFFReader.<init>(RIFFReader.java:61)
        at java.desktop/com.sun.media.sound.WaveExtensibleFileReader.getAudioFileFormatImpl(WaveExtensibleFileReader.java:172)
        at java.desktop/com.sun.media.sound.SunFileReader.getAudioFileFormat(SunFileReader.java:54)
        at java.desktop/com.sun.media.sound.WaveExtensibleFileReader.getAudioInputStream(WaveExtensibleFileReader.java:259)
        at java.desktop/javax.sound.sampled.AudioSystem.getAudioInputStream(AudioSystem.java:1010)
        at net.oijon.algonquin.App.createAudio(App.java:90)
        ... 1 more
```

**Comment by thetacola** [comment]
seems to be a windows console problem, might need to create a gui for it to work successfully

**Comment by thetacola** [comment]
fixed with 0.1.0
